packages/ui/__stories__/Dialog.stories.tsx

```diff
@@ -26,38 +25,20 @@ const DialogWithPortalFactory: FC<DialogProps> = ({ modalClassName, ...props })
       <Dialog
         {...props}
         modalClassName={cn(modalClassName, utilsStyles.modal)}
+        portalClassName={utilsStyles.modalPortal}
```

Comment:
@jankoritak shouldn't we add the `portalClassName` for all `Modals`? Why do we have to provide it for every story instead of having the default one in the `Modal` component?